docs/source/using/profiles.adoc

```diff
@@ -12,7 +12,26 @@ toc::[]
 == Overview
 
 {project} allows to create multiple instances of {project}.
-Each {project} instance can be created with different configurations and users can switch between different instances.
+Each {project} instance can be created with different configurations e.g memory, CPU, disk-size , add-ons etc and users can switch between different instances.
```

Comment:
There is a logical error here. So far we have not introduced 'profile' at all. Here we talk still "Each Minishift instance". Then next sentence is "Profile feature would help ...". At some stage, we need to say something like" an instance of the Minishift VM together with all its configuration and state information is called a Minsihift profile". 

Also I am missing the clarification that for now we have a "share nothing" architecture between profiles.